coa-helm/templates/cromwell.yaml

```diff
@@ -141,6 +141,7 @@ data:
             endpoint = "http://tes/v1/tasks"
             use_tes_11_preview_backend_parameters = true
             transform-blob-to-local-path = true
+	    workflow-execution-identity = "{{ .Values.config.workflowExecutionIdentity }}"
```

Comment:
Haha, I think Github is yelling at you about a tab character here. 

We'll also want to update `values.yaml` and `local_values.template.yaml` in this chart.